contracts/test/_trace.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Allows you to trace the execution of solidity call.
+ */
+const bre = require("@nomiclabs/buidler");
+
+// let tracingEnabled = false;
```

Comment:
this is pretty cool - assume it only works for the buidler vm right - not on a forked mainnet?

Reply:
Correct, just the buidler vm.